pkgs/build-support/cc-wrapper/utils.sh

```diff
@@ -24,26 +24,16 @@ badPath() {
 }
 
 expandResponseParams() {
-    local inparams=("$@")
-    local n=0
-    local p
-    params=()
-    while [ $n -lt ${#inparams[*]} ]; do
-        p=${inparams[n]}
-        case $p in
-            @*)
-                local parseResponseFile="@parseResponseFile@"
-                if [ -n "$parseResponseFile" ] ; then
-                    eval "params+=($("$parseResponseFile" "${p:1}"))"
-                else
-                    echo "Response files aren't supported during bootstrapping" >&2
-                    exit 1
-                fi
-                ;;
-            *)
-                params+=("$p")
-                ;;
-        esac
-        n=$((n + 1))
-    done
+    if [ -n "@expand-response-params@" ]; then
+        readarray -d '' params < <("@expand-response-params@" "$@")
```

Comment:
I have changed this to only call `expand-response-params` when `@FILE` args are present.

Thank you!